src/Antiforgery/src/Internal/DefaultAntiforgeryTokenStore.cs

```diff
@@ -57,8 +58,17 @@ public async Task<AntiforgeryTokenSet> GetRequestTokensAsync(HttpContext httpCon
             {
                 // Check the content-type before accessing the form collection to make sure
                 // we report errors gracefully.
-                var form = await httpContext.Request.ReadFormAsync();
-                requestToken = form[_options.FormFieldName];
+                try
+                {
+                    var form = await httpContext.Request.ReadFormAsync();
+                    requestToken = form[_options.FormFieldName];
+                }
+                catch (IOException ex)
```

Comment:
Could this catch an exception due to the body already having been read❔ May need to confirm this code path is hit only after making the body `Stream` re-windable.

Reply:
@Tratcher \ @halter73 what do you think? Should we inspect `RequestAborted` as part of this?

Reply:
I don't follow either of these comments.
- A request body that has already been read does not throw an IOException for additional reads, it returns 0 from Stream.ReadAsync.
- If the form has already been read then ReadFormAsync returns the cached form.
- If ReadFormAsync tries to read a body that's already been consumed I'd expect it to return an empty form, it can't tell the difference between a body that started empty and one that was consumed.

Reply:
The other exception worth catching here is InvalidDataException which is thrown when the form is invalid.

Reply:
> Should we inspect `RequestAborted` as part of this?

That's something I mentioned we can do if you're nervous about taking this change without that check, but I don't see a problem with assuming any and all IOExceptions signify aborted requests. I prefer the simplicity of treating all IOExceptions the same way.